.github/workflows/ci.yml

```diff
@@ -47,6 +49,28 @@ jobs:
     - name: Test projects
       run: dotnet test Microsoft-Perf-Tools-Linux-Android.sln --no-restore --verbosity normal
 
+    - name: Install PluginTool
+      run: dotnet tool install --global Microsoft.Performance.Toolkit.Plugins.Cli --version 0.1.25-preview      
+
+    - name: Create PTIX
+      run: |
+        $dirs = @{
+          "PerfDataExtension" = "PerfDataExtension/bin/Release/netstandard2.1/"
+          "PerfDataTxtExtension" = "PerfDataTxtExtension/bin/Release/netstandard2.1/"
+          "LTTngDataExtensions" = "LTTngDataExtensions/bin/Release/netstandard2.1/"
+          "LTTngDriver" = "LTTngDriver/bin/Release/netcoreapp8.0/"
+          "Perfetto" = "PerfettoCds/bin/Release/netstandard2.1/"
+          "Cloud-init" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/Cloud-init/bin/Release/netstandard2.1/"
+          "Dmesg" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/DmesgIsoLog/bin/Release/netstandard2.1/"
+          "WaLinuxAgent" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/WaLinuxAgent/bin/Release/netstandard2.1/"
+          "AndroidLogCat" = "LinuxLogParsers/LinuxPlugins-MicrosoftPerformanceToolkSDK/AndroidLogCat/bin/Release/netstandard2.1/"
+        }
+
+        foreach ($dir in $dirs.Keys) {
```

Comment:
The .ptix files also need to be copied to GH Workspace like the binary files are copied